src/pages/index.vue

```diff
@@ -1,127 +1,171 @@
 <template>
-  <main
-    class="flex h-screen flex-col justify-center gap-6 overflow-hidden bg-yellow lg:flex-row lg:gap-0"
-  >
-    <!-- TODO: Refine min-width for different breakpoints, remove magic numbers -->
-    <header
-      class="box-border flex w-full flex-grow flex-col justify-between lg:w-auto lg:min-w-[32rem] lg:justify-center xl:min-w-[64rem]"
+  <div class="index flex min-h-screen flex-col bg-yellow">
+    <!-- See layout `default.vue` for reference implementation. -->
+    <div v-if="isNewHeaderEnabled" class="sticky top-0 z-40 block">
```

Comment:
Can we remove the header and footer on the index page from this PR, leaving just the addition of the HomeGallery? Now that all pages have the same header and footer, we can simplify the layouts: use the `default` layout for all pages (including index and 404).  I was hoping to open a PR for moving `index.vue` and 404 pages to `default` layout after this PR is merged.